lib/openQAcoretest.pm

```diff
@@ -14,7 +14,7 @@ sub post_fail_hook {
     if (check_var('OPENQA_FROM_GIT', 1)) {
         send_key 'ctrl-c';     # Stop current command, if any
         assert_script_run 'cd /root/openQA';
-        enter_cmd 'script/openqa-cli api jobs';
+        enter_cmd '[ -e script/openqa-cli ] && script/openqa-cli api jobs';
```

Comment:
This will help if the git clone failed, but not if the `cpanm` following that fails.
I guess it's really not that easy.
I'm wondering if there is a general way to make it more obvious that a test is in the post_fail_hook stage.

Reply:
well, we can just split each test module to be in more specific steps. Then for each module we can define more specific post_fail_hooks